### PR TITLE
priy1289/featurecollectionlayer

### PR DESCRIPTION
**Pull request by priy1289**
Feature collection layer sample in android.

### CODE REVIEW

File: java/feature-collection-layer/README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Feature Collection Layer
+This sample demonstrates how to create a new Feature Collection with a Point, Polyline, and Polygon Feature Collection Table. The Collection is then displayed on the ArcGISMap as a Feature Collection Layer.
```
[the remark this answers is not in the document]
Comment (priy1289) [reply java/feature-collection-layer/README.md]: Ah, okay :)

**Comment by priy1289** [comment]
Thanks Trevor!

File: java/feature-collection-layer/README.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+* FeatureCollectionLayer
+* FeatureCollectionTable
+* Feature
+* Field
+* SimpleFillSymbol
+* SimpleLineSymbol
+* SimpleMarkerSymbol
+* SimpleRenderer
+
+###Tags
```
Comment (TADraeseke) [inline java/feature-collection-layer/README.md]: space missing between hashs and Tags

File: java/feature-collection-layer/src/main/java/com/esri/arcgisruntime/sample/featurecollectionlayer/MainActivity.java [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+package com.esri.arcgisruntime.sample.featurecollectionlayer;
+
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+
+import com.esri.arcgisruntime.data.Feature;
+import com.esri.arcgisruntime.data.FeatureCollection;
+import com.esri.arcgisruntime.data.FeatureCollectionTable;
+import com.esri.arcgisruntime.data.Field;
+import com.esri.arcgisruntime.geometry.*;
```
Comment (TADraeseke) [inline java/feature-collection-layer/src/main/java/com/esri/arcgisruntime/sample/featurecollectionlayer/MainActivity.java]: use single imports. Makes it clearer to our users which parts of our API are contained in the sample.

Comment (priy1289) [reply java/feature-collection-layer/src/main/java/com/esri/arcgisruntime/sample/featurecollectionlayer/MainActivity.java]: Done

**Comment by priy1289** [comment]
Thanks for the detailed review @TADraeseke :)

**Pull request merged**